### PR TITLE
Creating topic about documentation

### PR DESCRIPTION
**Pull request by olegario96**
It would be nice that new programmers read the documentation since the first contact with programming languages and IT stuff.

### CODE REVIEW

File: java/01-how-to-study.md [hunk 1]
```diff
@@ -136,3 +136,22 @@ Provavelmente não irá conseguir, mas se você estava estudando algo em Maio,
 pode lembrar que anotou algo a respeito na semana do dia 20. Essas coisas nem
 sempre pegam de uma semana para outra: às vezes você precisa revisar coisas que
 viu há meses ou mesmo anos).
+
+Leia a documentação
+-----
```
Comment (JPTIZ) [inline java/01-how-to-study.md]: Both the subtitle and "-" lines should be of the same length:

```
Like this
---------

Not like this
------
```

File: java/01-how-to-study.md [hunk 2]
```diff
@@ -136,3 +136,22 @@ Provavelmente não irá conseguir, mas se você estava estudando algo em Maio,
...
+
+Você provavelmente algum dia já foi até o Google e procurou "How to <sua dúvida
```
Comment (JPTIZ) [inline java/01-how-to-study.md]: For markdown, "\<text\>" means "text" is a link. Use "\\\<text\\\>" instead.

File: java/01-how-to-study.md [hunk 3]
```diff
@@ -136,3 +136,22 @@ Provavelmente não irá conseguir, mas se você estava estudando algo em Maio,
...
+aqui>". Esta é uma prática muito comum quando precisamos fazer alguma coisa 
+rápida. Geralmente encontramos a resposta facilmente em sites como Stack Overflow
+ou similares e em alguns casos até conseguimos boas respostas. O problema é que
+algumas vezes a pessoa que te deu a resposta está te passando uma solução que 
+não é a melhor de todas (e ele não está fazendo isso por mal, provavelmente, mas
```
Comment (JPTIZ) [inline java/01-how-to-study.md]: Actually "best solution of all" are too strong words. "an inadequate solution" or "bad solution" could fit better.

Also, when refering sites, prefer to link them:

```
em sites como [Stack Overflow](stackoverflow.com) ou similares
```

File: java/01-how-to-study.md [hunk 4]
```diff
@@ -136,3 +136,22 @@ Provavelmente não irá conseguir, mas se você estava estudando algo em Maio,
...
+sim porque ele também não leu a documentação e foi a maneira a qual encontrou
+para resolver o problema). Por isso sempre que tiver tempo, leia a documentação.
+Cada vez mais as linguagens e bibliotecas estão com a documentação muito rica
+e te dão passo-a-passo do que você pode fazer com a ferramenta. Dificilmente
+você não vai achar o que precisa na documentação e quando isso acontecer você
+mesmo pode contribuir para o documento ajudando o registro a ficar ainda mais
+atualizado e detalhando, contribuindo para futuros programadores que tiverem
```
Comment (JPTIZ) [inline java/01-how-to-study.md]: *detalhado

**Comment by ranisalt** [comment]
Top top PR

**Comment by olegario96** [comment]
@JPTIZ should be OK now

**Pull request merged**